src/webui/webapplication.h

```diff
@@ -35,6 +35,7 @@
 #include <QRegularExpression>
 #include <QSet>
 #include <QTranslator>
+#include <QDateTime>
```

Comment:
Why did you add this? Do you use it anywhere?

Reply:
That's strange... I'm also wondering why it's there.